styling/src/jsMain/kotlin/dev/fritz2/styling/theme/definitions.kt

```diff
@@ -120,50 +118,20 @@ class ZIndices(
         const val key: Property = "z-index: "
     }
 
-    /**
-     * [Property] for base z-index
-     */
-    val base: Property = "$baseValue"
-
-    /**
-     * [Property] for overlay z-index
-     */
-    val overlay: Property = "$overlayValue"
-
-    /**
-     * creates [Property] for a specific layer z-index
-     *
-     * Use self defined constants for the different layers of your UI.
-     *
-     * @param value number of layer the z-index should be calculated for
-     */
-    fun layer(value: Int): Property = zIndexFrom(layer, layerStep, value, 0)
-
-    /**
-     * creates [Property] for a specific toast z-index
-     *
-     * @param value number of toast the z-index should be calculated for
-     */
-    fun toast(value: Int): Property = zIndexFrom(toast, toastStep, value, 0)
+    val tableHeader: Property = tableHeader.toString()
+    val tooltip: Property = tooltip.toString()
+    val dropdown: Property = dropdown.toString()
+    val popover: Property = popover.toString()
+    val appFrame: Property = appFrame.toString()
+    val navbar: Property = navbar.toString()
+    val toast: Property = toast.toString()
+    val modal: Property = modal.toString()
 
-    /**
-     * creates [Property] for a specific modals z-index
-     *
-     * @param value number of modal the z-index should be calculated for
-     */
-    fun modal(value: Int): Property = zIndexFrom(modal, modalStep, value, 0)
-
-    /**
-     * creates [Property] for a specific modals z-index shifted by an offset
-     *
-     * @param value number of modal the z-index should be calculated for
-     * @param offset number to add to the final z-index in order to place an element below (negative value!) or on top
-     *               (positive value) of a regularly defined modal.
-     */
-    fun modal(value: Int, offset: Int): Property = zIndexFrom(modal, modalStep, value, offset)
+    operator fun Property.minus(value: Int): Property =
+        (this.toInt() - value).toString()
 
-    private fun zIndexFrom(level: Int, step: Int, value: Int, offset: Int) =
-        "${level + step * (value - 1) + offset}"
+    operator fun Property.plus(value: Int): Property =
```

Comment:
After further investigations it is absolutly necessary to rename those methods and stay away from operators: The calculations for the modal won't work correctly! It will rely to the built-in operator and do String concatenation instead...

I suggest to use ``above`` and ``below``.

Reply:
After discussion we come up with `infix fun Property.raisedBy(value: Int): Property` for both cases (negative `value` for lowering).